elyra/cli/pipeline_app.py

```diff
@@ -128,6 +117,21 @@ def _preprocess_pipeline(pipeline_path, runtime, runtime_config, work_dir):
     return pipeline_definition
 
 
+def _execute_pipeline(pipeline_definition):
+    try:
+        with yaspin(text="Processing pipeline ..."):
+            # parse pipeline
+            pipeline_object = PipelineParser().parse(pipeline_definition)
+            # process pipeline
+            asyncio.get_event_loop().run_until_complete(PipelineProcessorManager.instance().process(pipeline_object))
```

Comment:
I haven't tested this yet, but `local` mode has a progress bar when executing nodes, so we might get weird rending issues if we use a spinner on top of it.

Reply:
tested, `Processing pipeline ...` and the progress bar fight over each other. I think we should only render the spinner for `submit` and not for `run`. I also think "Submitting pipeline..." would make more sense

Reply:
I thought about this, but you will see sort of a "hang" for a long time if you are executing non-papermill things such as long python or R script.

Reply:
ah, I see, should we add some sort of spinner/progressbar internally for non-papermill executions?

Reply:
maybe it's better to just drop the spinner and use a simple `click.echo()`, that way it won't obstruct the papermill progress bars, but we still get the indication that it's submitting/running something

Reply:
click.echo() will not help on long-running scripts.

Reply:
@lresende I agree that a spinner would be preferable, but as you also mentioned in another comment the scripts can print things and it gobbles the progress bar that is much more useful than a simple spinner. I think a static echo is an okay compromise until we have a better way to handle spinners and outputs together

Reply:
@bourdakos1 Please checkout the covid-notebook repo and try running/submitting `covid-notebooks/pipelines/us_data.pipeline` and see if the static waiting is acceptable. The spinner at least gives some sense of something going on while the user wait a long task.

Reply:
yea, it's not ideal, but I think I'm okay with this for now. When using `run` it shows nice progress bars for each notebook being run (I know it won't show anything for regular scripts). When using `submit` it doesn't take incredibly long to complete, an animation is nice, but `...` also lets me know it might take a minute. We could also add more details saying it might take a moment to finish.

Eventually we can use the blanket spinner for `submit` and rely on the papermill logs for executing notebooks and add our own logging for scripts, unless that is something you want to add to this PR